zkg

```diff
@@ -315,6 +402,9 @@ def create_manager(args, config):
 
         raise
 
+    pkg_dependencies = load_pkg_dependencies(manager)
+    manager.set_pkg_dependencies(pkg_dependencies)
```

Comment:
Look like something manager should be doing in its own `__init__`

Reply:
Fixed.